packages/client/tests/functional/composites/selection/prisma/_schema.ts

```diff
@@ -20,11 +20,16 @@ export default testMatrix.setupSchema(({ provider }) => {
   type Profile {
     name Name
     url String
+    favoriteThings Thing[]
   }
   
   type Name {
     firstName String
     lastName String
   }
+
+  type Thing {
+    name String
+  }
```

Comment:
I'm wondering if we should also test like in https://github.com/prisma/prisma/blob/c6982ebf3d860444370f4b835b604fda94b92089/packages/client/src/__tests__/integration/happy/exhaustive-schema-mongo/schema.prisma#L37
For a composite type in a composite type

Reply:
I think only optional composite was missing? Added tests for that too.